modules/run-consul/run-consul

```diff
@@ -309,6 +313,17 @@ EOF
 )
   fi
 
+  local connect_configuration=""
+  if [[ "$enable_connect" == "true" && "$server" == "true" ]]; then
```

Comment:
`enable_connect` needs to be declared above and passed explicitly into the function. We use `local` rather than global variables.

Reply:
Takled in #202.